.github/ISSUE_TEMPLATE/label-request.yml

```
@@ -8,24 +8,24 @@ body:
   - type: markdown
     attributes:
       value: |
-         **Main label changelog issue**:  
-        [List of current and previous labels](https://github.com/wg-lux/endoreg-db/issues/102)
+        **Main label changelog**:  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)